quickjs.h

```diff
@@ -327,7 +350,8 @@ JS_EXTERN JSRuntime *JS_NewRuntime(void);
 JS_EXTERN void JS_SetRuntimeInfo(JSRuntime *rt, const char *info);
 /* use 0 to disable memory limit */
 JS_EXTERN void JS_SetMemoryLimit(JSRuntime *rt, size_t limit);
-JS_EXTERN void JS_SetDumpFlags(JSRuntime *rt, uint64_t flags);
+JS_EXTERN int JS_SetDumpFlags(JSRuntime *rt, uint64_t flags);
```

Comment:
We could leave it as it was and make it a noop in release buids.

Reply:
Should users be able to tell if they've linked to a build that supports dumping?

Reply:
Currently, due to the very specific nature of it, the user knows for sure.

Reply:
Changed to noop